lib/redis_client/ruby_connection.rb

```diff
@@ -148,6 +148,7 @@ def connect
       )
       true
     rescue SystemCallError, OpenSSL::SSL::SSLError, SocketError => error
+      socket.close if socket
```

Comment:
```suggestion
      socket&.close
```

Seems like you angered rubocop...